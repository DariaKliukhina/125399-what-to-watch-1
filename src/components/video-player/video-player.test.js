import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const mock = {
  film: {
    title: `Bohemian Rhapsody`,
    picture: `poster.jpg`,
    preview: `video.mp4`,
  },
};

it(`Video player correctly renders`, () => {
  const props = {
    videoSrc: mock.film.preview,
    posterSrc: mock.film.picture,
    options: {
      width: 100,
      height: 100,
    }
  };

  const tree = renderer.create(<VideoPlayer {...props}/>).toJSON();

  expect(tree).toMatchSnapshot();
});

