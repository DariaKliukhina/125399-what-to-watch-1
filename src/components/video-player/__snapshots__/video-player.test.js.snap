// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The application is displayed correctly. VideoPlayer correctly renders after launch 1`] = `
<video
  height={100}
  loop={true}
  muted={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width={100}
/>
`;
